bazel/repository_locations.bzl

```diff
@@ -834,13 +834,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Chrome URL parsing library",
         project_desc = "Chrome URL parsing library",
         project_url = "https://quiche.googlesource.com/googleurl",
-        # Static snapshot of https://quiche.googlesource.com/quiche/+archive/ef0d23689e240e6c8de4c3a5296b209128c87373.tar.gz.
-        version = "ef0d23689e240e6c8de4c3a5296b209128c87373",
-        sha256 = "d769283fed1319bca68bae8bdd47fbc3a7933999329eee850eff1f1ea61ce176",
+        # Static snapshot of https://quiche.googlesource.com/quiche/+archive/561705e0066ff11e6cb97b8092f1547835beeb92.tar.gz.
```

Comment:
Bringing the discussion https://github.com/envoyproxy/envoy/pull/17794#discussion_r694213267. re: moving to GH.

@danzh2010 @RenjieTang @moderation I think we need to move to GitHub, but that needs Google's decision? Since it will be hosted along with https://github.com/google/quiche (or if it is still only all about envoy, judging from the name: "envoy-integration", probably we may host it on envoyproxy org)? cc. @htuch @yanavlasov.

Reply:
I agree we should move this similar to what we did with quiche. I'm not sure what is involved in doing this.

Reply:
I think @DavidSchinazi @bencebeky helped out here for QUICHE. So they might also have some idea here.

Reply:
The is an internal Google approval process for third party libraries.  After that, Google's GitHub admin creates the repo under https://github.com/google/.  Reach out to me though an internal channel if you decide to go down this route.  At https://github.com/envoyproxy/envoy/pull/17794#discussion_r711386718 @dio mentions hosting it under https://github.com/envoyproxy/, that might be a better fit for this purpose.

Reply:
Ultimately Google owns this code so we will need someone on the Google side to drive this. Between hosting it within the Envoy org or within a Google org I have no preference, as long as there is a single canonical source. We can quickly make you a repo within the Envoy org if you like. @yanavlasov is this something your team can help drive?